nbextensions/bulkImportCell/bulkImportCell.js

```diff
@@ -362,8 +368,9 @@ define([
                         inputs: typesToFiles,
                         params: initialParams,
                         app: {
-                            fileParamIds: fileParamIds,
-                            otherParamIds: otherParamIds,
+                            fileParamIds,
+                            otherParamIds,
+                            outputParamIds,
```

Comment:
Track the output parameter ids for each app separately from the other types of parameter.